synapse/module_api/__init__.py

```diff
@@ -111,6 +112,12 @@ def __init__(self, hs: "HomeServer", auth_handler):
         self._spam_checker = hs.get_spam_checker()
         self._account_validity_handler = hs.get_account_validity_handler()
         self._third_party_event_rules = hs.get_third_party_event_rules()
+        self._presence_router = hs.get_presence_router()
+
+    #################################################################################
+    # Constants for modules to ues
+
+    PRESENCE_ALL_USERS = PresenceRouter.ALL_USERS
```

Comment:
I really think this would make more sense to be exposed in `__all__` directly rather than as a property of the `ModuleApi`.